addons/point_of_sale/static/src/css/pos_receipts.css

```diff
@@ -37,6 +42,10 @@
     font-size: 75%;
 }
 
+.pos-receipt-order-data p {
+    margin-bottom: 0;
+}
+
```

Comment:
what's this used for?

Reply:
![image](https://github.com/odoo/odoo/assets/157357943/d3a3ff91-68d4-452e-b550-251a7a93a286)
the odoo point of sale line had space below it this was to remove that space 😄

Reply:
makes sense, you could also use bootstrap for this 
![image](https://github.com/odoo/odoo/assets/118446179/49e38db4-2033-446c-8f65-4a62c6f2a313)
To me, the utility classes help because you can have a good idea what a page will look like just by looking at the html. With custom css classes it's not always the case